src/promptflow/promptflow/batch/_base_executor_proxy.py

```diff
@@ -115,6 +117,17 @@ async def exec_aggregation_async(
         result = self._process_http_response(response)
         return AggregationResult.deserialize(result)
 
+    async def ensure_executor_startup(self, error_file):
```

Comment:
Maybe add this method to `APIBasedProxy`? Since you need to send a http request to check healthy.